csv2mtx.py

```diff
@@ -57,7 +57,9 @@ def __iter__(self):
                 else:
                     self.y_axis_values.append(row[0])
                     for col, value in enumerate(row[1:]):
-                        if float(value):
+                        value = value.strip()
+                        assert ' ' not in value, f'Non-numeric value "{value}" at col {col}'
```

Comment:
The error message is misleading but I realize now that the `float` cast below makes this entire assertion unnecessary.